core/domain/exp_domain.py

```diff
@@ -1396,6 +1396,37 @@ def get_translatable_text(self, language_code):
 
         return state_names_to_content_id_mapping
 
+    def get_completed_translations(self, language_code):
+        """Returns all the contents along with the corresponding
+        translations for which the process of the process of translation in
+        the given language has been completed.
+
+        Args:
+            language_code: str. The language code in which translation is
+                required.
+
+        Returns:
+            dict(str, list(str)). A dict where the key is type of data
```

Comment:
This isn't correct, right? We're returning a named tuple here. 

Also, it seems strange that we're returning two lists here and the client has to assume that they are ordered perfectly so that the indices match up. In the design doc, we wanted to return a list of tuple-like objects, not a tuple of lists so that we maintain the content to translation mapping.